stdlib/psych/0/dbm.rbs

```diff
@@ -1,5 +1,5 @@
 %a{annotate:rdoc:skip}
-module YAML
+module Psych
```

Comment:
Note: I renamed `YAML::DBM` and `YAML::Store` to `Psych::DBM` and `Psych::Store`. It is needed to define the `YAML` module as an alias for the `Psych` module in RBS.

Accurately, these definitions are not correct because the implementation in Ruby is placed under the `YAML` module, not the `Psych` module.
https://github.com/ruby/ruby/blob/master/lib/yaml/dbm.rb

But they are recognized as `Psych::*` in runtime:

```
irb(main):001> require 'yaml/dbm'
=> true
irb(main):002> YAML
=> Psych
irb(main):003> YAML::DBM
=> Psych::DBM
irb(main):004> 
```

So this is not so wrong, I think.